openshift/_python36.dc.json

```diff
@@ -99,6 +99,13 @@
     {
       "name": "REPLICA_MAX",
       "value": "1"
+    },
+    {
+      "name": "ENVIRONMENT_NAME",
+      "displayName": "Environment Name (Environment Id)",
+      "description": "The name or Id of the environment.  This variable is used by the webhook integration to identify the environment in which the backup notifications originate.",
+      "required": false,
```

Comment:
Can you put in a default value here?
Usually if something is not required we need to give it a default value or vice versa.

Reply:
Will do, this is copied from your database backup though so you might need one there too.

Reply:
Good catch! Can you update that in your PR? If not, im happy to do in one of my follow ups :)